### ISSUE TITLE
Checkbox values not being set when using 0.9.5

### ISSUE THREAD

**Issue by dnroot** [vkononov/fillable-pdf#27]
Using the following PDF form:
[claim-template.pdf](https://github.com/vkononov/fillable-pdf/files/9965520/claim-template.pdf)

And setting the following fields:
```ruby
pdf.set_fields({
  "assignment.yes".to_sym => "Yes",
  "relation.self".to_sym => "Yes",
  "insurance_type.7".to_sym => "Yes",
  "sex.male".to_sym => "Yes"
})
```
We expect the checkboxes corresponding to the above fields to be checked.

In 0.9.4, they were checked:
<img width="498" alt="image" src="https://user-images.githubusercontent.com/4641416/200684243-1143722d-8504-433d-a9ee-304c898764e7.png">

In 0.9.5, they are not:
<img width="499" alt="image" src="https://user-images.githubusercontent.com/4641416/200684300-0b7687ba-b35a-49af-b2a2-397aba588173.png">

**Comment by thomasgallagheres** [comment]
I've also experienced this behaviour, thought that the API to check the boxes had changed but that was not the case, rolling back to 0.9.3 with the same calling code works.

**Comment by vkononov** [comment]
Thanks for reporting this @dnroot and @thomasgallagheres. Yes, it definitely has to do with this line: `pdf_field(key).setValue(value.to_s, generate_appearance)`. I'll take a look at it soon and will likely make the generate_appearance flag optional, just for those who need it. @dnroot, thanks for sharing the PDF - that will really help with debugging!

**Comment by vkononov** [comment]
This issue has been fixed in [v0.9.5.1](https://github.com/vkononov/fillable-pdf/releases/tag/v0.9.5.1).
